apps/health_campaign_field_worker_app/lib/pages/checklist/checklist_eligibility_assessment.dart

```diff
@@ -186,10 +186,10 @@ class _EligibilityChecklistViewPageState
                               ineligibilityReasons,
                             );
                             var descriptionText = ifIneligible
-                                ? '"Ineligible"'
+                                ? 'the beneficiary is ineligible for SPAQ administration'
                                 : ifReferral
-                                    ? '"Referral"'
-                                    : '"Administration"';
+                                    ? 'You are being redirected to the beneficiary referral screen'
+                                    : 'You are being redirected to SPAQ administration screen';
```

Comment:
@yash-egov @amitverma-egov why do we have hardcoded string why there is no lcoalisations